pkg/otlp/map_provider.go

```diff
@@ -3,6 +3,7 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2021-present Datadog, Inc.
 
+//go:build !serverless
 // +build !serverless
```

Comment:
Is this unrelated?

Reply:
Argh, my code editor adds this automatically since I have Go 1.17 installed. It's unrelated but it should be harmless.

Reply:
Removed by using another editor